.github/workflows/cross-bootstrap-tools.yml

```diff
@@ -2,9 +2,9 @@ name: Cross-build Kernel
 
 on:
   push:
-    branches: [ master, dev ]
+    branches: [ main, dev ]
```

Comment:
I don't think we need to reference all three since we will keep master and main in sync in the short term.